In [1]:
import pandas as pd
import matplotlib.pyplot as mat
import seaborn as sea

#### Importing the libraries pandas ,matplotlib.pyplot an seaborn from python, in order to perform column operations on the dataset.   

In [2]:
brand=pd.read_csv('G:/Adidas Vs Nike - (Tamil).csv')

In [3]:
brand

Product Name  Product ID  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      Listing Price  Sale Price  Discount                    Brand  \
0             14999        7499        50  Adidas Adidas ORIGINALS   
1              7599        3799        50         Adidas ORIGINALS   
2               999         599        40        Adidas CORE / NEO   
3              6999        3499        50        Adidas CORE / NEO   
4              7999        3999        50         Adidas ORIGINALS   
...             ...         ...       ...                      ...   
3263          15995       12797         0                     Nike   
3264           4995        3497         0                     Nike   
3265           8495        5947         0                     Nike   
3266              0       16995         0                     Nike   
3267           8995        6297         0                     Nike   

                                            Description  Rating  Reviews  \
0     Channeling the streamlined look of an '80s rac...     4.8       41   
1     A modern take on adidas sport heritage, tailor...     3.3       24   
2     These adidas Puka slippers for women's come wi...     2.6       37   
3     Inspired by modern tech runners, these women's...     4.1       35   
4     This design is inspired by vintage Taekwondo s...     3.5       72   
...                                                 ...     ...      ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...     5.0        1   
3264  The Nike Phantom Venom Club IC is engineered f...     0.0        0   
3265  The soft upper of the Nike Mercurial Superfly ...     5.0        1   
3266  The Nike Air Max 98 features the OG design lin...     4.0        4   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...     0.0        0   

             Last Visited  
0     2020-04-13T15:06:14  
1     2020-04-13T15:06:15  
2     2020-04-13T15:06:15  
3     2020-04-13T15:06:15  
4     2020-04-13T15:06:15  
...                   ...  
3263  2020-04-13T15:41:01  
3264  2020-04-13T15:41:03  
3265  2020-04-13T15:41:07  
3266  2020-04-13T15:41:19  
3267  2020-04-13T15:42:57  

[3268 rows x 10 columns]

In [4]:
brand.columns

Index(['Product Name', 'Product ID', 'Listing Price', 'Sale Price', 'Discount',
       'Brand', 'Description', 'Rating', 'Reviews', 'Last Visited'],
      dtype='object')

#### Displaying the variable names in the dataset.

In [5]:
brand['total']=brand['Listing Price']+brand['Sale Price']

#### Added a new variable in the dataset by combining the variables Listing price and Sale price. 

In [6]:
brand

Product Name  Product ID  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      Listing Price  Sale Price  Discount                    Brand  \
0             14999        7499        50  Adidas Adidas ORIGINALS   
1              7599        3799        50         Adidas ORIGINALS   
2               999         599        40        Adidas CORE / NEO   
3              6999        3499        50        Adidas CORE / NEO   
4              7999        3999        50         Adidas ORIGINALS   
...             ...         ...       ...                      ...   
3263          15995       12797         0                     Nike   
3264           4995        3497         0                     Nike   
3265           8495        5947         0                     Nike   
3266              0       16995         0                     Nike   
3267           8995        6297         0                     Nike   

                                            Description  Rating  Reviews  \
0     Channeling the streamlined look of an '80s rac...     4.8       41   
1     A modern take on adidas sport heritage, tailor...     3.3       24   
2     These adidas Puka slippers for women's come wi...     2.6       37   
3     Inspired by modern tech runners, these women's...     4.1       35   
4     This design is inspired by vintage Taekwondo s...     3.5       72   
...                                                 ...     ...      ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...     5.0        1   
3264  The Nike Phantom Venom Club IC is engineered f...     0.0        0   
3265  The soft upper of the Nike Mercurial Superfly ...     5.0        1   
3266  The Nike Air Max 98 features the OG design lin...     4.0        4   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...     0.0        0   

             Last Visited  total  
0     2020-04-13T15:06:14  22498  
1     2020-04-13T15:06:15  11398  
2     2020-04-13T15:06:15   1598  
3     2020-04-13T15:06:15  10498  
4     2020-04-13T15:06:15  11998  
...                   ...    ...  
3263  2020-04-13T15:41:01  28792  
3264  2020-04-13T15:41:03   8492  
3265  2020-04-13T15:41:07  14442  
3266  2020-04-13T15:41:19  16995  
3267  2020-04-13T15:42:57  15292  

[3268 rows x 11 columns]

In [7]:
brand['low']=brand['Listing Price']/brand['Sale Price']

In [8]:
brand

Product Name  Product ID  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      Listing Price  Sale Price  Discount                    Brand  \
0             14999        7499        50  Adidas Adidas ORIGINALS   
1              7599        3799        50         Adidas ORIGINALS   
2               999         599        40        Adidas CORE / NEO   
3              6999        3499        50        Adidas CORE / NEO   
4              7999        3999        50         Adidas ORIGINALS   
...             ...         ...       ...                      ...   
3263          15995       12797         0                     Nike   
3264           4995        3497         0                     Nike   
3265           8495        5947         0                     Nike   
3266              0       16995         0                     Nike   
3267           8995        6297         0                     Nike   

                                            Description  Rating  Reviews  \
0     Channeling the streamlined look of an '80s rac...     4.8       41   
1     A modern take on adidas sport heritage, tailor...     3.3       24   
2     These adidas Puka slippers for women's come wi...     2.6       37   
3     Inspired by modern tech runners, these women's...     4.1       35   
4     This design is inspired by vintage Taekwondo s...     3.5       72   
...                                                 ...     ...      ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...     5.0        1   
3264  The Nike Phantom Venom Club IC is engineered f...     0.0        0   
3265  The soft upper of the Nike Mercurial Superfly ...     5.0        1   
3266  The Nike Air Max 98 features the OG design lin...     4.0        4   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...     0.0        0   

             Last Visited  total       low  
0     2020-04-13T15:06:14  22498  2.000133  
1     2020-04-13T15:06:15  11398  2.000263  
2     2020-04-13T15:06:15   1598  1.667780  
3     2020-04-13T15:06:15  10498  2.000286  
4     2020-04-13T15:06:15  11998  2.000250  
...                   ...    ...       ...  
3263  2020-04-13T15:41:01  28792  1.249902  
3264  2020-04-13T15:41:03   8492  1.428367  
3265  2020-04-13T15:41:07  14442  1.428451  
3266  2020-04-13T15:41:19  16995  0.000000  
3267  2020-04-13T15:42:57  15292  1.428458  

[3268 rows x 12 columns]

In [9]:
brand.rename(columns={'Sale Price':'Sales'})

Product Name  Product ID  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      Listing Price  Sales  Discount                    Brand  \
0             14999   7499        50  Adidas Adidas ORIGINALS   
1              7599   3799        50         Adidas ORIGINALS   
2               999    599        40        Adidas CORE / NEO   
3              6999   3499        50        Adidas CORE / NEO   
4              7999   3999        50         Adidas ORIGINALS   
...             ...    ...       ...                      ...   
3263          15995  12797         0                     Nike   
3264           4995   3497         0                     Nike   
3265           8495   5947         0                     Nike   
3266              0  16995         0                     Nike   
3267           8995   6297         0                     Nike   

                                            Description  Rating  Reviews  \
0     Channeling the streamlined look of an '80s rac...     4.8       41   
1     A modern take on adidas sport heritage, tailor...     3.3       24   
2     These adidas Puka slippers for women's come wi...     2.6       37   
3     Inspired by modern tech runners, these women's...     4.1       35   
4     This design is inspired by vintage Taekwondo s...     3.5       72   
...                                                 ...     ...      ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...     5.0        1   
3264  The Nike Phantom Venom Club IC is engineered f...     0.0        0   
3265  The soft upper of the Nike Mercurial Superfly ...     5.0        1   
3266  The Nike Air Max 98 features the OG design lin...     4.0        4   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...     0.0        0   

             Last Visited  total       low  
0     2020-04-13T15:06:14  22498  2.000133  
1     2020-04-13T15:06:15  11398  2.000263  
2     2020-04-13T15:06:15   1598  1.667780  
3     2020-04-13T15:06:15  10498  2.000286  
4     2020-04-13T15:06:15  11998  2.000250  
...                   ...    ...       ...  
3263  2020-04-13T15:41:01  28792  1.249902  
3264  2020-04-13T15:41:03   8492  1.428367  
3265  2020-04-13T15:41:07  14442  1.428451  
3266  2020-04-13T15:41:19  16995  0.000000  
3267  2020-04-13T15:42:57  15292  1.428458  

[3268 rows x 12 columns]

#### Renamed a 'Sale Price' variable as 'Sales'.

In [10]:
brand.rename(columns={'Listing Price':'Original Price','total':'final'})

Product Name  Product ID  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      Original Price  Sale Price  Discount                    Brand  \
0              14999        7499        50  Adidas Adidas ORIGINALS   
1               7599        3799        50         Adidas ORIGINALS   
2                999         599        40        Adidas CORE / NEO   
3               6999        3499        50        Adidas CORE / NEO   
4               7999        3999        50         Adidas ORIGINALS   
...              ...         ...       ...                      ...   
3263           15995       12797         0                     Nike   
3264            4995        3497         0                     Nike   
3265            8495        5947         0                     Nike   
3266               0       16995         0                     Nike   
3267            8995        6297         0                     Nike   

                                            Description  Rating  Reviews  \
0     Channeling the streamlined look of an '80s rac...     4.8       41   
1     A modern take on adidas sport heritage, tailor...     3.3       24   
2     These adidas Puka slippers for women's come wi...     2.6       37   
3     Inspired by modern tech runners, these women's...     4.1       35   
4     This design is inspired by vintage Taekwondo s...     3.5       72   
...                                                 ...     ...      ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...     5.0        1   
3264  The Nike Phantom Venom Club IC is engineered f...     0.0        0   
3265  The soft upper of the Nike Mercurial Superfly ...     5.0        1   
3266  The Nike Air Max 98 features the OG design lin...     4.0        4   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...     0.0        0   

             Last Visited  final       low  
0     2020-04-13T15:06:14  22498  2.000133  
1     2020-04-13T15:06:15  11398  2.000263  
2     2020-04-13T15:06:15   1598  1.667780  
3     2020-04-13T15:06:15  10498  2.000286  
4     2020-04-13T15:06:15  11998  2.000250  
...                   ...    ...       ...  
3263  2020-04-13T15:41:01  28792  1.249902  
3264  2020-04-13T15:41:03   8492  1.428367  
3265  2020-04-13T15:41:07  14442  1.428451  
3266  2020-04-13T15:41:19  16995  0.000000  
3267  2020-04-13T15:42:57  15292  1.428458  

[3268 rows x 12 columns]

#### Renamed Two variables 'Listing price' as 'Original Price' and 'total' as 'final'.

In [11]:
brand.drop(['Reviews','Rating'],axis=1)

Product Name  Product ID  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      Listing Price  Sale Price  Discount                    Brand  \
0             14999        7499        50  Adidas Adidas ORIGINALS   
1              7599        3799        50         Adidas ORIGINALS   
2               999         599        40        Adidas CORE / NEO   
3              6999        3499        50        Adidas CORE / NEO   
4              7999        3999        50         Adidas ORIGINALS   
...             ...         ...       ...                      ...   
3263          15995       12797         0                     Nike   
3264           4995        3497         0                     Nike   
3265           8495        5947         0                     Nike   
3266              0       16995         0                     Nike   
3267           8995        6297         0                     Nike   

                                            Description         Last Visited  \
0     Channeling the streamlined look of an '80s rac...  2020-04-13T15:06:14   
1     A modern take on adidas sport heritage, tailor...  2020-04-13T15:06:15   
2     These adidas Puka slippers for women's come wi...  2020-04-13T15:06:15   
3     Inspired by modern tech runners, these women's...  2020-04-13T15:06:15   
4     This design is inspired by vintage Taekwondo s...  2020-04-13T15:06:15   
...                                                 ...                  ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...  2020-04-13T15:41:01   
3264  The Nike Phantom Venom Club IC is engineered f...  2020-04-13T15:41:03   
3265  The soft upper of the Nike Mercurial Superfly ...  2020-04-13T15:41:07   
3266  The Nike Air Max 98 features the OG design lin...  2020-04-13T15:41:19   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...  2020-04-13T15:42:57   

      total       low  
0     22498  2.000133  
1     11398  2.000263  
2      1598  1.667780  
3     10498  2.000286  
4     11998  2.000250  
...     ...       ...  
3263  28792  1.249902  
3264   8492  1.428367  
3265  14442  1.428451  
3266  16995  0.000000  
3267  15292  1.428458  

[3268 rows x 10 columns]

#### We have dropped the variables 'Ratings' and 'Reviews' from the dataset.

In [12]:
brand.columns=[col.upper()for col in brand.columns]

In [13]:
brand

PRODUCT NAME  PRODUCT ID  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      LISTING PRICE  SALE PRICE  DISCOUNT                    BRAND  \
0             14999        7499        50  Adidas Adidas ORIGINALS   
1              7599        3799        50         Adidas ORIGINALS   
2               999         599        40        Adidas CORE / NEO   
3              6999        3499        50        Adidas CORE / NEO   
4              7999        3999        50         Adidas ORIGINALS   
...             ...         ...       ...                      ...   
3263          15995       12797         0                     Nike   
3264           4995        3497         0                     Nike   
3265           8495        5947         0                     Nike   
3266              0       16995         0                     Nike   
3267           8995        6297         0                     Nike   

                                            DESCRIPTION  RATING  REVIEWS  \
0     Channeling the streamlined look of an '80s rac...     4.8       41   
1     A modern take on adidas sport heritage, tailor...     3.3       24   
2     These adidas Puka slippers for women's come wi...     2.6       37   
3     Inspired by modern tech runners, these women's...     4.1       35   
4     This design is inspired by vintage Taekwondo s...     3.5       72   
...                                                 ...     ...      ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...     5.0        1   
3264  The Nike Phantom Venom Club IC is engineered f...     0.0        0   
3265  The soft upper of the Nike Mercurial Superfly ...     5.0        1   
3266  The Nike Air Max 98 features the OG design lin...     4.0        4   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...     0.0        0   

             LAST VISITED  TOTAL       LOW  
0     2020-04-13T15:06:14  22498  2.000133  
1     2020-04-13T15:06:15  11398  2.000263  
2     2020-04-13T15:06:15   1598  1.667780  
3     2020-04-13T15:06:15  10498  2.000286  
4     2020-04-13T15:06:15  11998  2.000250  
...                   ...    ...       ...  
3263  2020-04-13T15:41:01  28792  1.249902  
3264  2020-04-13T15:41:03   8492  1.428367  
3265  2020-04-13T15:41:07  14442  1.428451  
3266  2020-04-13T15:41:19  16995  0.000000  
3267  2020-04-13T15:42:57  15292  1.428458  

[3268 rows x 12 columns]

#### Displayed the Variables name of the dataset in the Uppercase.

In [14]:
brand.columns=[col.lower()for col in brand.columns]

In [15]:
brand

product name  product id  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      listing price  sale price  discount                    brand  \
0             14999        7499        50  Adidas Adidas ORIGINALS   
1              7599        3799        50         Adidas ORIGINALS   
2               999         599        40        Adidas CORE / NEO   
3              6999        3499        50        Adidas CORE / NEO   
4              7999        3999        50         Adidas ORIGINALS   
...             ...         ...       ...                      ...   
3263          15995       12797         0                     Nike   
3264           4995        3497         0                     Nike   
3265           8495        5947         0                     Nike   
3266              0       16995         0                     Nike   
3267           8995        6297         0                     Nike   

                                            description  rating  reviews  \
0     Channeling the streamlined look of an '80s rac...     4.8       41   
1     A modern take on adidas sport heritage, tailor...     3.3       24   
2     These adidas Puka slippers for women's come wi...     2.6       37   
3     Inspired by modern tech runners, these women's...     4.1       35   
4     This design is inspired by vintage Taekwondo s...     3.5       72   
...                                                 ...     ...      ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...     5.0        1   
3264  The Nike Phantom Venom Club IC is engineered f...     0.0        0   
3265  The soft upper of the Nike Mercurial Superfly ...     5.0        1   
3266  The Nike Air Max 98 features the OG design lin...     4.0        4   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...     0.0        0   

             last visited  total       low  
0     2020-04-13T15:06:14  22498  2.000133  
1     2020-04-13T15:06:15  11398  2.000263  
2     2020-04-13T15:06:15   1598  1.667780  
3     2020-04-13T15:06:15  10498  2.000286  
4     2020-04-13T15:06:15  11998  2.000250  
...                   ...    ...       ...  
3263  2020-04-13T15:41:01  28792  1.249902  
3264  2020-04-13T15:41:03   8492  1.428367  
3265  2020-04-13T15:41:07  14442  1.428451  
3266  2020-04-13T15:41:19  16995  0.000000  
3267  2020-04-13T15:42:57  15292  1.428458  

[3268 rows x 12 columns]

#### Displayed the Variables name of the dataset in the lowercase.

In [16]:
brand.sort_values(by=['listing price','sale price'],ascending=[True,False])

product name  product id  listing price  \
3148           Nike Air VaporMax Flyknit 2 LXX  AT5673-700              0   
2693     Nike Mercurial Superfly 7 Elite SE FG  CT1099-093              0   
2931    Nike Mercurial Superfly 7 Elite MDS FG  BQ5469-703              0   
3149     Nike Mercurial Superfly 7 Elite SE FG  CT1099-093              0   
3164    Nike Mercurial Superfly 7 Elite MDS FG  BQ5469-703              0   
...                                        ...         ...            ...   
1553   Men's adidas Originals ZX 4000 4D Shoes      EF9624          27999   
1563   Men's adidas Originals ZX 4000 4D Shoes      EF9623          27999   
1642  Unisex adidas Running Alphaedge 4D Shoes      FV6106          27999   
2554   Men's adidas Originals ZX 4000 4D Shoes      BD7927          29999   
2579   Men's adidas Originals ZX 4000 4D Shoes      BD7865          29999   

      sale price  discount                     brand  \
3148       36500         0                      Nike   
2693       24995         0                      Nike   
2931       24995         0                      Nike   
3149       24995         0                      Nike   
3164       24995         0                      Nike   
...          ...       ...                       ...   
1553       27999         0          Adidas ORIGINALS   
1563       27999         0          Adidas ORIGINALS   
1642       27999         0  Adidas SPORT PERFORMANCE   
2554       29999         0          Adidas ORIGINALS   
2579       29999         0          Adidas ORIGINALS   

                                            description  rating  reviews  \
3148  The Nike Air VaporMax Flyknit 2 LXX places you...     0.0        0   
2693  Everyone starts somewhere and Kylian Mbappé st...     0.0        0   
2931  Dream of speed and play fast in the Nike Mercu...     5.0        1   
3149  Everyone starts somewhere and Kylian Mbappé st...     0.0        0   
3164  Dream of speed and play fast in the Nike Mercu...     5.0        1   
...                                                 ...     ...      ...   
1553  The ZX trainers series resets the running scen...     3.5       18   
1563  The ZX trainers series resets the running scen...     3.1       62   
1642  Add an edge to your game with the new AlphaEdg...     2.2       53   
2554  In the '80s, the science of sport and wearable...     2.4       37   
2579  In the '80s, the science of sport and wearable...     4.0       53   

             last visited  total  low  
3148  2020-04-13T15:36:42  36500  0.0  
2693  2020-04-13T15:28:15  24995  0.0  
2931  2020-04-13T15:32:30  24995  0.0  
3149  2020-04-13T15:36:43  24995  0.0  
3164  2020-04-13T15:36:47  24995  0.0  
...                   ...    ...  ...  
1553  2020-04-13T15:07:16  55998  1.0  
1563  2020-04-13T15:07:16  55998  1.0  
1642  2020-04-13T15:07:21  55998  1.0  
2554  2020-04-13T15:07:57  59998  1.0  
2579  2020-04-13T15:07:57  59998  1.0  

[3268 rows x 12 columns]

#### In this we have sorted two variables 'Listing price' and 'Sale Price' and then we have arranged the variable 'Listing Price' in the ascending order.

In [17]:
brand.sort_values(by='total',ascending=True)

product name product id  \
2044  Men's adidas Swimming Beach Print Maxout Slippers     CL0233   
2420                     Men's adidas Toe Side Slippers     CL0252   
147                   Women's adidas Swim Jung Slippers     CM0072   
2046         Men's adidas Swim Eezay Maxout II Slippers     CM0106   
2493         Men's adidas Swim Eezay Maxout II Slippers     CM0105   
...                                                 ...        ...   
1438            Men's adidas Originals ZX 4000 4D Shoes     EF9625   
1642           Unisex adidas Running Alphaedge 4D Shoes     FV6106   
1553            Men's adidas Originals ZX 4000 4D Shoes     EF9624   
2554            Men's adidas Originals ZX 4000 4D Shoes     BD7927   
2579            Men's adidas Originals ZX 4000 4D Shoes     BD7865   

      listing price  sale price  discount                     brand  \
2044            899         449        50         Adidas CORE / NEO   
2420            999         499        50         Adidas CORE / NEO   
147             999         499        50         Adidas CORE / NEO   
2046            999         499        50         Adidas CORE / NEO   
2493            999         499        50         Adidas CORE / NEO   
...             ...         ...       ...                       ...   
1438          27999       27999         0          Adidas ORIGINALS   
1642          27999       27999         0  Adidas SPORT PERFORMANCE   
1553          27999       27999         0          Adidas ORIGINALS   
2554          29999       29999         0          Adidas ORIGINALS   
2579          29999       29999         0          Adidas ORIGINALS   

                                            description  rating  reviews  \
2044  These adidas Beach Print Maxout 2 slippers for...     3.1       36   
2420  The adidas casual sandals for men. Features st...     3.8       22   
147   These adidas Jung slippers for women come with...     3.2       20   
2046  These casual adidas slippers for men come with...     5.0        1   
2493  These casual adidas slippers for men come with...     2.7       87   
...                                                 ...     ...      ...   
1438  Thousands of athletes. Years of data. adidas 4...     2.8       32   
1642  Add an edge to your game with the new AlphaEdg...     2.2       53   
1553  The ZX trainers series resets the running scen...     3.5       18   
2554  In the '80s, the science of sport and wearable...     2.4       37   
2579  In the '80s, the science of sport and wearable...     4.0       53   

             last visited  total       low  
2044  2020-04-13T15:07:38   1348  2.002227  
2420  2020-04-13T15:07:52   1498  2.002004  
147   2020-04-13T15:06:24   1498  2.002004  
2046  2020-04-13T15:07:38   1498  2.002004  
2493  2020-04-13T15:07:54   1498  2.002004  
...                   ...    ...       ...  
1438  2020-04-13T15:07:11  55998  1.000000  
1642  2020-04-13T15:07:21  55998  1.000000  
1553  2020-04-13T15:07:16  55998  1.000000  
2554  2020-04-13T15:07:57  59998  1.000000  
2579  2020-04-13T15:07:57  59998  1.000000  

[3268 rows x 12 columns]

#### Sorted the variable named 'total' and arranged its values in ascending order.

In [18]:
brand.sort_values(by='total')

product name product id  \
2044  Men's adidas Swimming Beach Print Maxout Slippers     CL0233   
2420                     Men's adidas Toe Side Slippers     CL0252   
147                   Women's adidas Swim Jung Slippers     CM0072   
2046         Men's adidas Swim Eezay Maxout II Slippers     CM0106   
2493         Men's adidas Swim Eezay Maxout II Slippers     CM0105   
...                                                 ...        ...   
1438            Men's adidas Originals ZX 4000 4D Shoes     EF9625   
1642           Unisex adidas Running Alphaedge 4D Shoes     FV6106   
1553            Men's adidas Originals ZX 4000 4D Shoes     EF9624   
2554            Men's adidas Originals ZX 4000 4D Shoes     BD7927   
2579            Men's adidas Originals ZX 4000 4D Shoes     BD7865   

      listing price  sale price  discount                     brand  \
2044            899         449        50         Adidas CORE / NEO   
2420            999         499        50         Adidas CORE / NEO   
147             999         499        50         Adidas CORE / NEO   
2046            999         499        50         Adidas CORE / NEO   
2493            999         499        50         Adidas CORE / NEO   
...             ...         ...       ...                       ...   
1438          27999       27999         0          Adidas ORIGINALS   
1642          27999       27999         0  Adidas SPORT PERFORMANCE   
1553          27999       27999         0          Adidas ORIGINALS   
2554          29999       29999         0          Adidas ORIGINALS   
2579          29999       29999         0          Adidas ORIGINALS   

                                            description  rating  reviews  \
2044  These adidas Beach Print Maxout 2 slippers for...     3.1       36   
2420  The adidas casual sandals for men. Features st...     3.8       22   
147   These adidas Jung slippers for women come with...     3.2       20   
2046  These casual adidas slippers for men come with...     5.0        1   
2493  These casual adidas slippers for men come with...     2.7       87   
...                                                 ...     ...      ...   
1438  Thousands of athletes. Years of data. adidas 4...     2.8       32   
1642  Add an edge to your game with the new AlphaEdg...     2.2       53   
1553  The ZX trainers series resets the running scen...     3.5       18   
2554  In the '80s, the science of sport and wearable...     2.4       37   
2579  In the '80s, the science of sport and wearable...     4.0       53   

             last visited  total       low  
2044  2020-04-13T15:07:38   1348  2.002227  
2420  2020-04-13T15:07:52   1498  2.002004  
147   2020-04-13T15:06:24   1498  2.002004  
2046  2020-04-13T15:07:38   1498  2.002004  
2493  2020-04-13T15:07:54   1498  2.002004  
...                   ...    ...       ...  
1438  2020-04-13T15:07:11  55998  1.000000  
1642  2020-04-13T15:07:21  55998  1.000000  
1553  2020-04-13T15:07:16  55998  1.000000  
2554  2020-04-13T15:07:57  59998  1.000000  
2579  2020-04-13T15:07:57  59998  1.000000  

[3268 rows x 12 columns]

#### We have sorted the variable named 'total'. 

In [19]:
brand.drop(['low'],axis=1,inplace=True)

#### We have dropped the variable named 'low' and used inplace function.  

In [20]:
brand

product name  product id  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      listing price  sale price  discount                    brand  \
0             14999        7499        50  Adidas Adidas ORIGINALS   
1              7599        3799        50         Adidas ORIGINALS   
2               999         599        40        Adidas CORE / NEO   
3              6999        3499        50        Adidas CORE / NEO   
4              7999        3999        50         Adidas ORIGINALS   
...             ...         ...       ...                      ...   
3263          15995       12797         0                     Nike   
3264           4995        3497         0                     Nike   
3265           8495        5947         0                     Nike   
3266              0       16995         0                     Nike   
3267           8995        6297         0                     Nike   

                                            description  rating  reviews  \
0     Channeling the streamlined look of an '80s rac...     4.8       41   
1     A modern take on adidas sport heritage, tailor...     3.3       24   
2     These adidas Puka slippers for women's come wi...     2.6       37   
3     Inspired by modern tech runners, these women's...     4.1       35   
4     This design is inspired by vintage Taekwondo s...     3.5       72   
...                                                 ...     ...      ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...     5.0        1   
3264  The Nike Phantom Venom Club IC is engineered f...     0.0        0   
3265  The soft upper of the Nike Mercurial Superfly ...     5.0        1   
3266  The Nike Air Max 98 features the OG design lin...     4.0        4   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...     0.0        0   

             last visited  total  
0     2020-04-13T15:06:14  22498  
1     2020-04-13T15:06:15  11398  
2     2020-04-13T15:06:15   1598  
3     2020-04-13T15:06:15  10498  
4     2020-04-13T15:06:15  11998  
...                   ...    ...  
3263  2020-04-13T15:41:01  28792  
3264  2020-04-13T15:41:03   8492  
3265  2020-04-13T15:41:07  14442  
3266  2020-04-13T15:41:19  16995  
3267  2020-04-13T15:42:57  15292  

[3268 rows x 11 columns]

In [21]:
brand.groupby('sale price').count()

product name  product id  listing price  discount  brand  \
sale price                                                             
449                    1           1              1         1      1   
499                    4           4              4         4      4   
599                    8           8              8         8      8   
600                    1           1              1         1      1   
649                    3           3              3         3      3   
...                  ...         ...            ...       ...    ...   
23999                  5           5              5         5      5   
24995                  4           4              4         4      4   
27999                  4           4              4         4      4   
29999                  2           2              2         2      2   
36500                  1           1              1         1      1   

            description  rating  reviews  last visited  total  
sale price                                                     
449                   1       1        1             1      1  
499                   4       4        4             4      4  
599                   8       8        8             8      8  
600                   1       1        1             1      1  
649                   3       3        3             3      3  
...                 ...     ...      ...           ...    ...  
23999                 5       5        5             5      5  
24995                 4       4        4             4      4  
27999                 4       4        4             4      4  
29999                 2       2        2             2      2  
36500                 1       1        1             1      1  

[227 rows x 10 columns]

#### By Performing groupby function on Sale Price variable and used count function it is displays us this variable takes place in 227 observations.  

In [28]:
brand=pd.read_csv('G:/Adidas Vs Nike - (Tamil).csv')

In [29]:
brand

Product Name  Product ID  \
0     Women's adidas Originals NMD_Racer Primeknit S...      AH2430   
1                  Women's adidas Originals Sleek Shoes      G27341   
2                     Women's adidas Swim Puka Slippers      CM0081   
3      Women's adidas Sport Inspired Questar Ride Shoes      B44832   
4              Women's adidas Originals Taekwondo Shoes      D98205   
...                                                 ...         ...   
3263                                 Air Jordan 8 Retro  CI1236-100   
3264                         Nike Phantom Venom Club IC  AO0578-717   
3265               Nike Mercurial Superfly 7 Academy TF  AT7978-414   
3266                                    Nike Air Max 98  AH6799-300   
3267                                     Nike P-6000 SE  CJ9585-600   

      Listing Price  Sale Price  Discount                    Brand  \
0               NaN        7499        50  Adidas Adidas ORIGINALS   
1               NaN        3799        50         Adidas ORIGINALS   
2             999.0         599        40        Adidas CORE / NEO   
3            6999.0        3499        50        Adidas CORE / NEO   
4            7999.0        3999        50         Adidas ORIGINALS   
...             ...         ...       ...                      ...   
3263        15995.0       12797         0                     Nike   
3264         4995.0        3497         0                     Nike   
3265         8495.0        5947         0                     Nike   
3266            0.0       16995         0                     Nike   
3267         8995.0        6297         0                     Nike   

                                            Description  Rating  Reviews  \
0     Channeling the streamlined look of an '80s rac...     4.8       41   
1     A modern take on adidas sport heritage, tailor...     3.3       24   
2     These adidas Puka slippers for women's come wi...     2.6       37   
3     Inspired by modern tech runners, these women's...     4.1       35   
4     This design is inspired by vintage Taekwondo s...     3.5       72   
...                                                 ...     ...      ...   
3263  The Air Jordan 8 Retro recaptures the memorabl...     5.0        1   
3264  The Nike Phantom Venom Club IC is engineered f...     0.0        0   
3265  The soft upper of the Nike Mercurial Superfly ...     5.0        1   
3266  The Nike Air Max 98 features the OG design lin...     4.0        4   
3267  A mash-up of Pegasus' past, the Nike P-6000 SE...     0.0        0   

             Last Visited  
0     2020-04-13T15:06:14  
1     2020-04-13T15:06:15  
2     2020-04-13T15:06:15  
3     2020-04-13T15:06:15  
4     2020-04-13T15:06:15  
...                   ...  
3263  2020-04-13T15:41:01  
3264  2020-04-13T15:41:03  
3265  2020-04-13T15:41:07  
3266  2020-04-13T15:41:19  
3267  2020-04-13T15:42:57  

[3268 rows x 10 columns]

In [30]:
print(brand['Listing Price'].isnull().sum())

2


#### We have found out that there are two null values present in the 'Listing Price'' variable. 

In [31]:
print(brand.columns.values)

['Product Name' 'Product ID' 'Listing Price' 'Sale Price' 'Discount'
 'Brand' 'Description' 'Rating' 'Reviews' 'Last Visited']


#### Displayed the variables but by using print function.